auto_round/mllm/template.py

```diff
@@ -22,6 +22,15 @@
 from .processor import BasicProcessor, PROCESSORS
 
 TEMPLATES: Dict[str, "Template"] = {}
+SUPPORT_ONLY_TEXT_MODELS = [
+    "phi3_v",
+    "cogvlm2",
+    "llava",
+    "qwen2_vl",
+    "deepseek_vl_v2",
+    "chatglm",
```

Comment:
no any postfix in chatglm?

Reply:
No, only chatglm
https://huggingface.co/THUDM/glm-4v-9b/blob/6a1215eea198982bcfc2a639312471b9ac1ac044/config.json#L3